zhaquirks/tuya/tuya_smoke.py

```diff
@@ -76,16 +82,59 @@ def _update_attribute(self, attrid, value):
 (
     TuyaQuirkBuilder("_TZE200_aycxwiau", "TS0601")
     .applies_to("_TZE200_dq1mfjug", "TS0601")
-    .applies_to("_TZE200_m9skfctm", "TS0601")
-    .applies_to("_TZE200_rccxox8p", "TS0601")
-    .applies_to("_TZE284_rccxox8p", "TS0601")
     .applies_to("_TZE200_vzekyi4c", "TS0601")
     .applies_to("_TZE204_vawy74yh", "TS0601")
     .tuya_smoke(dp_id=1)
     .skip_configuration()
     .add_to_registry()
 )
 
+(
+    TuyaQuirkBuilder("_TZE284_rccxox8p", "TS0601")
+    .applies_to("_TZE200_m9skfctm", "TS0601")
+    .applies_to("_TZE200_rccxox8p", "TS0601")
+    .tuya_smoke(dp_id=1)
+    .tuya_sensor(
+        dp_id=2,
+        type=t.int16s,
+        attribute_name="smoke_concentration",
+        translation_key="smoke_concentration",
+        fallback_name="Smoke concentration",
```

Comment:
Minor thing, but just to be consistent with other v2 quirks, can we move `translation_key` and `fallback_name` to the bottom in all v2 quirk methods, so below `unit`.